### DISCUSSION TITLE
Anschlüsse bündeln?

### DISCUSSION THREAD

**Discussion by micw** [hoylabs/OpenDTU-OnBattery#883] [General]
Seh ich es richtig, dass ich z.B. beim HM-1200 bei einem Eingang auf ~600 Watt (11,5A * 50V) für die Entnahme aus der Batterie beschränkt bin? Oder kann ich einfach mehrere Eingänge bündeln und ~1200W aus der Batterie entnehmen?

**Comment by micw** [comment]
Ich habe dazu noch gefunden:
* https://github.com/helgeerbe/OpenDTU-OnBattery/discussions/285
* https://github.com/helgeerbe/OpenDTU-OnBattery/discussions/244

Schlussfolgerung: wird laut Hoymiles nicht empfohlen, funktioniert aber einwandfrei.

**Reply by Manos1966** [reply]
> Seh ich es richtig, dass ich z.B. beim HM-1200 bei einem Eingang auf ~600 Watt (11,5A * 50V) für die Entnahme aus der Batterie beschränkt bin? Oder kann ich einfach mehrere Eingänge bündeln und ~1200W aus der Batterie entnehmen?

Ja und Ja

...und gleiche Laenge der Kabel (vor allem minus Kabel) nicht vergessen!
https://github.com/helgeerbe/OpenDTU-OnBattery/wiki/Kabell%C3%A4nge%28n%29-zwischen-Batterie-und-Wechselrichter

**Reply by Farquarl** [reply]
Kann ich bestätigen, meine Kabel sind nicht gleich lang und es liegen immer geringfügig andere Wattzahlen an, obwohl alles vom Busbar kommt.

**Reply by micw** [reply]
Danke. Was mir aus den verlinkten Dokumenten nicht ganz klar ist: hat es "ernsthafte" Konsequenzen, wenn die Leitungen einen geringfügig unterschiedlichen ohmschen Widerstand haben?

**Reply by Farquarl** [reply]
Also bei mir nicht, es werden zwar bei den strings unterschiedliche leistungen angezeigt aber das was kombiniert aus zwei victrons rauskommt geht nach Effizienz Abzügen auch ins netzt (oder batterie). Ich denke da kommt es auch auf deine batterie bzw den laderegler an. Ich habe ein 36volt system (3x12v 50ah powerqueen in serie) Ansonsten ist man immer am unteren limit des hoymiles (22v) Das gibt dann wieder Effizienz Einbußen. Durch die etwas höhere Spannung komme ich dann nicht an die 10A grenze pro string des wechselrichters. Ich kann mir vorstellen das es bei einem 24v system schon eher passieren kann.

**Reply by Manos1966** [reply]
> Danke. Was mir aus den verlinkten Dokumenten nicht ganz klar ist: hat es "ernsthafte" Konsequenzen, wenn die Leitungen einen geringfügig unterschiedlichen ohmschen Widerstand haben?

"ernsthafte" Konsequenzen hat bis heute (Gott sei Dank) niemand bis jetzt berichtet.

Es gab Berichte wo Leute versucht haben zu verstehen, warum die einkommende DC Strom-Staerke Amps nicht mit der ausgehende gleich war. Und teils davon liegt an die Batterie Spannung 24V haben staerkere Abweichungen als 48V usw usw.

**Reply by micw** [reply]
Alles klar, vielen Dank für die Infos. Ich habe selbst aktuell einen Multiplus 2 mit Pylontech UC5000 am laufen. Derzeit tausche ich mich mit einem Freund aus, der sowas "in klein" möchte, daher schaue ich mir das Projekt hier in dem Kontext an. Den Multiplus 2 kann ich leider nicht empfehlen, da die Geräte irrsinnig laut sind (technisch aber toll).
